docs/0.3.5-alpha/stdlib/doc.md

```diff
@@ -0,0 +1,52 @@
+## Builtin vs StdLib
+
+[Builtins](/advanced_syntax/builtins) are method that are included in the Amber compiler and don't need to be imported in the code.
+
+In contrast, the standard library (stdlib) is a collection of Amber functions that are embedded in every Amber release. Each version of Amber may include changes to the standard library and you need to import these functions in your code. These functions are more advanced and can accept various parameters.
+
+## Standard library and Shellcheck
+
+Just like the Amber's compiled Bash output, all standard library functions are built from the ground up to be shellcheck compliant. This means that you can focus more on building the logic and spend less time on keeping the code predictable and valid.
+
+## How to use it
+
+Below is an example of how to use the standard library to generate documentation (using the [script](https://github.com/amber-lang/amber-docs/sync-stdlib-doc.ab) provided on the Amber Documentation repository):
+
+```ab
+import { download } from "std/http"
+import { split, contains } from "std/text"
+import { file_exist } from "std/fs"
+
+unsafe $rm -fr /tmp/amber-git$
```

Comment:
Consider adding inner spaces to Bash fragments in examples, i.e. change `$...$` to `$ ... $`, for consistency.